packages/playground-examples/copy/ja/3-7/Fixits/Const to let.ts

```diff
@@ -0,0 +1,9 @@
+//// { compiler: {  }, order: 1 }
+
+// 3.7では、const変数の値が再代入されたときに、素早くletに変換する機能が追加されました。
+
+// 以下のエラーをハイライトして、クイックフィックスを実行することで試すことができます。
```

Comment:
IMO: このhighlighting、popupを表示することを指している可能性がある気がしますが、原文をリスペクトしておきます
```suggestion
// ハイライトされている以下のエラーから、クイックフィックスを実行することで試すことができます。
```